components/global/global_header.tsx

```diff
@@ -95,7 +96,17 @@ const GlobalHeader = (): JSX.Element | null => {
         };
     }, [enabled]);
 
-    if (!enabled || !isLoggedIn) {
+    console.log('[GLOBAL HEADER] isMobileView', isMobileView);
+
+    useEffect(() => {
+        console.log('[GLOBAL HEADER] general useEffect: isMobileView', isMobileView);
+    });
+
+    useEffect(() => {
+        console.log('[GLOBAL HEADER] limited isMobileView', isMobileView);
+    }, [isMobileView]);
+
```

Comment:
testing? :P

Reply:
To demonstrate the effect of dispatching the windowSize action from the window resize event on the rendering of components that subscribe to the selectors. I'm going to remove if we decide to move forward with this.

Reply:
Removing now that everyone has had a chance to look.